Packs/Rapid7InsightVMCloud/Integrations/Rapid7InsightVMCloud/Rapid7InsightVMCloud.py

```diff
@@ -0,0 +1,472 @@
+
+    Args:
+        client (Client Object): Client object with request.
+        scan_id : The identifier of the scan.
+
+    """
+
+    endpoint = f"/v4/integration/scan/{scan_id}"
+    method = 'GET'
+    response = client.make_request(
+        method=method,
+        url_suffix=endpoint
+    )
+    if response:
+        response_data = response
+        markdown = tableToMarkdown(
+            'This is Required Scan Information',
+            response_data
+        )
+        result = CommandResults(
+            readable_output=markdown,
+            outputs_prefix='Rapid7.InsighVMCloud.Scans',
+            outputs_key_field='id',
+            outputs=response_data
+        )
+        return result
+    else:
+        return_error('no response')
+
+
+def get_scan_engines_command(client: Client, page: int, size: int) -> CommandResults:
+    """
+    Retrieves a page of scan engines.
+
+    Args:
+        client (Client Object): Client object with request.
+        page: The index of the page (zero-based) to retrieve
+        size: The number of records per page to retrieve.
+
+    """
+
+    args = demisto.args()
+    method = "GET"
+    params = {
+        "page": args.get("page"),
+        "size": args.get("size")
+    }
+    res = int(params.get('page'))
+    endpoint = f"/v4/integration/scan/engine"
+    response = client.make_request(method=method, url_suffix=endpoint, params=params)
+    if int(args.get("size")) > 500:
+        return_error("Exceed size limit")
+    else:
+        response = client.make_request(
+            method=method,
+            url_suffix=endpoint,
+            params=params
+        )
+        if response:
+            response_data = response.get("data")
+            markdown = tableToMarkdown(
+                'This is Required Scan Information',
+                response_data
+            )
+            result = CommandResults(
+                readable_output=markdown,
+                outputs_prefix='Rapid7.InsighVMCloud.Engines',
+                outputs_key_field='id',
+                outputs=response_data
+            )
+            return result
+        else:
+            return_error('no response')
+
+
+def start_scan_command(client: Client, asset_id: str, name: str) -> CommandResults:
+    """
+    Starts a scan.
+
+    Args:
+        client (Client Object): Client object with request.
+        asset_id : The identifiers of the assets to scan.
+        name : The name of the scan.
+
+    """
+
+    args = demisto.args()
+    assets_id = args.get("asset_id")
+    list_id = assets_id.split(',')
+    method = "POST"
+    endpoint = "/v4/integration/scan"
+    data = {
+        "asset_ids": list_id,
+        "name": name
+    }
+
+    response = client.make_request(
+        method=method,
+        url_suffix=endpoint,
+        json_data=data
+    )
+    if response:
+        scan_response = response.get("scans")
+        headers = ["id", "engine_id", "asset_ids", "name"]
```

Comment:
You created the headers variable but you don't use it, what do you need it for ?

Reply:
Thanks for review, I have resolved this issue.